src/config/dark-sites.config

```diff
@@ -518,6 +519,7 @@ playcode.io/$
 playerdb.co
 pleroma.site
 pleroma.social
+plex.doorcraft.de
```

Comment:
Please remove this entry.
![image](https://user-images.githubusercontent.com/25481501/108500394-9b90e000-72a7-11eb-86bb-bc3de66150f3.png)

Reply:
lmao, its dark if you login (google for plex webinterface)
do you only want sites on the list that are dark without having to login or sites that are restricted to only a small userbase?